test/commands/break_test.rb

```diff
@@ -19,31 +19,36 @@ def program
         12:      end
         13:    end
         14:
-        15:    y = 3
-        16:
-        17:    byebug
+        15:    def self.#{example_module_method}
```

Comment:
I've refactored the breakpoint tests to make them a bit more resilient, so you don't have to change other tests just to add this one. I'll push the changes soon.

Once I do that, rewriting the test should be easy, but if you don't feel like working on this over and over again I can adapt it and credit you in the commit message. Up to you!

Thanks @x-yuri, this is a nice fix!